securedrop/source_templates/index.html

```diff
@@ -14,8 +14,8 @@
     {% endassets %}
   </head>
   <body>
-    <div class="js-warning warning">{{ gettext('<strong>We recommend turning the Security Slider to High to protect your anonymity:</strong>') }} <a id="disable-js" href="">{{ gettext('Learn how to set it to high') }}</a>{{ gettext(', or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
-    <div class="use-tor-browser warning">{{ gettext('<strong>We recommend using Tor Browser to access SecureDrop:</strong>') }} <a id="recommend-tor" href="{{ url_for("recommend_tor_browser") }}">{{ gettext('Learn how to install it') }}</a>{{ gettext(', or ignore this warning to continue.') }} <img id="use-tor-browser-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
+    <div class="js-warning warning">{{ gettext('<strong>We recommend turning the Security Slider to High to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to high</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
+    <div class="use-tor-browser warning">{{ gettext('<strong>We recommend using Tor Browser to access SecureDrop:</strong> <a id="recommend-tor" href="{tor_browser_url}">Learn how to install it</a>, or ignore this warning to continue.').format(tor_browser_url=url_for('recommend_tor_browser')) }} <img id="use-tor-browser-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
```

Comment:
OK for use-tor-browser-warning

![source-index_javascript](https://user-images.githubusercontent.com/433594/30684097-30e66082-9eb0-11e7-9f51-7c3a0129ddc5.png)

Manually verified with a tor browser than the js-warning-warning string renders correctly and that the image suggesting the security slider change shows.